front/src/modules/ui/table/components/EntityTableHeaderOptions.tsx

```diff
@@ -83,6 +84,7 @@ export const EntityTableHeaderOptions = ({
             </StyledDropdownMenuItemsContainer>
           </StyledDropdownMenu>
         }
+        dropdownHotkeyScope={{ scope: TableHotkeyScope.Table }}
```

Comment:
Thank you for your contribution!

Although, we need to create a new HotkeyScope for this, as `TableHotkeyScope.Table` will catch hotkeys specific to the table.

For instance: in `EntityTable`, you can see that a hotkey for `"escape"` has been set for the `Table` HotkeyScope, that resets table row selection. But when the dropdown is open, `"escape"` should close the dropdown instead.

If you use this scope for the dropdown, there will be conflicts between the table and dropdown hotkeys.

The new scope could be named `TableHotkeyScope.ColumnHeaderDropdown` for instance.